### PR TITLE
docs: Sample performance metrics 

### LINKED ISSUE #409: Default resource limits
The sample deployment should probably have resource limits set. 

Possibly in the future a guide/recommendation for setting limits. 

Thanks to @rosskukulinski these 2 issues might help:
https://github.com/heptio/gimbal/issues/72
https://github.com/heptio/gimbal/pull/122

### PR DESCRIPTION
**Pull request by stevesloka**
Fixes #409 by adding a sample test run results across a varying number of service/ingressroutes and load to Envoy running within a sample AWS environment. 

The sample doc is only a guideline across many different sets of test run criteria. 

Given the results, I came up with these sample resource constraints that could apply to a generic "catch-all" type environment:

**_Envoy:_**
```yaml
       resources:
          requests:
            memory: "256Mi"
            cpu: 1
          limits:
            memory: "512Mi"
            cpu: 2
```

**_Contour:_**
```yaml
     resources:
          requests:
            memory: "100Mi"
            cpu: "500m"
          limits:
            memory: "256Mi"
            cpu: 1
```

Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: docs/resource_limits.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      - 10 GB Network
+  - Contour
+    - Single Instance
+      - 4 Instances of Envoy running in a Daemonset
+      - Each instance of Envoy is running with HostNetwork
+    - Cluster Network Bandwidth
+
+Having a good understanding of the available bandwidth is key when it comes to analyzing performance. It will give you a sense of how many requests per second you can expect to push through the network you are working with.
+
+Use iperf3 to figure out the bandwidth available between two of the kubernetes nodes. The following will deploy an iperf3 server on one node, and an iperf3 client on another node:
```
Comment (youngnick) [inline docs/resource_limits.md]: Should there be a command here?

**Comment by youngnick** [comment]
Default requests look good too. Maybe we'll need a note/comment about them only being starting points in the examples though?

File: docs/resource_limits.md (hunk 1 above)
Comment (stevesloka) [reply docs/resource_limits.md]: Ahh good call, I stole this from some other notes on how I set up the perf testing. I'll fix up.

**Comment by stevesloka** [comment]
@youngnick good point on including the samples in the doc! I was hesitant to add to the deployment/yaml files, so this is a good place for them.

**Comment by youngnick** [comment]
Double LGTM, awesome.

**Pull request merged**